cucumber/authenticators_jwt/features/authn_status_jwt.feature

```diff
@@ -899,3 +899,253 @@ Feature: JWT Authenticator - Status Check
     When I GET "/authn-jwt/raw/cucumber/status"
     Then the HTTP response status code is 500
     And the authenticator status check fails with error "CONJ00037E Missing value for resource: cucumber:variable:conjur/authn-jwt/raw/audience>"
+
+  Scenario: ONYX-10875: mapping-claims configured but secret not populated, 500 Error
+    Given I load a policy:
+    """
+    - !policy
+      id: apps
+      body:
+      - !host myuser
+
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !webservice
+        annotations:
+          description: Authentication service for JWT tokens, based on raw JWKs.
+
+      - !variable
+        id: jwks-uri
+
+      - !variable
+        id: token-app-property
+
+      - !variable
+        id: mapping-claims
+
+      - !group users
+
+      - !permit
+        role: !group users
+        privilege: [ read, authenticate ]
+        resource: !webservice
+
+      - !webservice
+        id: status
+        annotations:
+          description: Status service to check that the authenticator is configured correctly
+
+      - !group
+          id: operators
+          annotations:
+            description: Group of users who can check the status of the authenticator
+
+      - !permit
+        role: !group operators
+        privilege: [ read ]
+        resource: !webservice status
+
+    - !user alice
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/operators
+      member:
+      - !user alice
+    """
+    And I am the super-user
+    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
+    And I successfully set authn-jwt "token-app-property" variable to value "user"
+    And I login as "alice"
+    And I save my place in the log file
+    When I GET "/authn-jwt/raw/cucumber/status"
+    Then the HTTP response status code is 500
+    And the authenticator status check fails with error "CONJ00037E Missing value for resource: cucumber:variable:conjur/authn-jwt/raw/mapping-claims>"
+
+  Scenario: ONYX-10876: enforced-claims configured but secret not populated, 500 Error
```

Comment:
why its not 502?

Reply:
Acctually it should be 500 i fix the tests